.github/ISSUE_TEMPLATE/DOCS.yaml

```
@@ -0,0 +1,53 @@
+name: "Feature Request"
+description: Create an issue for documentation changes
+title: "Docs: <title>"
+labels: ["docs", "triage required"]
```

```suggestion
labels: ["docs"]
```
Do we need `triage required` on all the template types?